addons/l10n_it_edi_ndd/models/account_move.py

```diff
@@ -1,4 +1,5 @@
 from odoo.addons.l10n_it_edi_ndd.models.account_payment_methode_line import L10N_IT_PAYMENT_METHOD_SELECTION
+from odoo.tools.sql import column_exists, create_column
```

Comment:
I suppose you put it here because of the misplaced `from odoo.addons....` import ? The base `from odoo import ...` should always be the first of the odoo imports so could you either put yours after it or even put both yours and the `odoo.addons....` one after it depending on whether you want to leave that line untouched or not ? Thanks :+1: